### PR TITLE
add flag "autoupdated" to transaction changelog entries

### PR DESCRIPTION
**Pull request by joka**
Internal changes:

- add flag 'autoupdated' to ChangeLogMetadata tuples (transaction changelog) and IResourceModified/IItemNewVersionAdded events. 
  It is True if the resource creation or modification was caused by autoupdateing versionable resources. This allows us to filter entries without changed content (For example  new rate versions triggerd by a new proposal version).

### CODE REVIEW

File: src/adhocracy_core/adhocracy_core/events/__init__.py [hunk 1]
```diff
@@ -64,7 +64,8 @@ class ResourceSheetModified:
     :param registry: pyramid.registry.Registry
     :param old_appstruct: The old :term:`appstruct` data (dict)
     :param new_appstruct: The new :term:`appstruct` data (dict)
-    :param request: The current request or None
```
Comment (2e2a) [inline src/adhocracy_core/adhocracy_core/events/__init__.py]: request is still a parameter

**Comment by 2e2a** [comment]
@joka Please review last commits setting autoupdated in additional places.

**Comment by joka** [comment]
Units tests are missing and exiting ones are failing

**Comment by joka** [comment]
@2e2a can you have a quick look at the last commit, then merge?

**Comment by 2e2a** [comment]
lgtm

**Pull request merged**